docs/introduction.mdx

```diff
@@ -9,56 +9,122 @@ import SolanaIcon from '@site/src/assets/SolanaIcon'
+  title="Polkadot" 
+  subtitle="stDOT" 
+  Icon={<PolkadotIcon />} 
+/> 
+ 
+</Grid> 
+ 
 
-<GridLink
-  to="https://docs.polygon.lido.fi"
-  title="Polygon"
-  subtitle="stMATIC"
-  Icon={<PolygonIcon />}
-/>
+## Liquid staking 
 
-<GridLink
-  to="https://docs.cosmos.lido.fi"
-  title="Cosmos"
-  subtitle="stAtom"
-  Icon={<CosmosIcon />}
-/>
+### Problem statement
 
-<GridLink
-  to="https://docs.kusama.lido.fi"
-  title="Kusama"
-  subtitle="stKSM"
-  Icon={<KusamaIcon />}
-/>
+Traditionally, staking in Proof-of-Stake (PoS) protocol based projects has been about locking one’s assets in one project for a long time and expecting a fixed, predetermined staking reward in return. While it guarantees the return on staked assets much like a bond, it also limits the opportunities of generating higher returns on those assets from the DeFi ecosystem. If you’ve staked all of your crypto holdings, you can’t invest or trade in more profitable crypto pairs on exchanges. 
 
-</Grid>
+### Solution 
+ 
+Liquid staking allows using the staked crypto assets in other trading or investing opportunities to let the user get the best of both worlds - a reward on your staked assets, as well as the returns from new trading/investing opportunities. Liquid staking introduce various fundamental benefits by: 
+- Allowing users to earn staking rewards without fully locking their `st[token]`;  
+- Making it possible to earn rewards on as small a deposit as users want (i.e, Ethereum requires minimum 32 `ETH` staked);  
+- Providing the `st[token]` a building block for other applications and protocols (e.g., as collateral in lending or other trading DeFi solutions);  
+- Providing an alternative to exchange staking, solo staking, and other semi-custodial and decentralised protocols. 
+ 
+### Advantages over solo staking and exchange staking 
+ 
+**Solo staking** is great, but it comes with some disadvantages. Setting up a validator node requires a pristine technical understanding; brings with it a minimum deposit of 32 ETH; slashing and offline penalties can get very severe if the staking is managed improperly; the staked amount is locked up for a significant period. Eventually, it leads to a small number of stakers, which is bad for any PoS ecosystem. 
+ 
+Alternatively, it may be possible to produce staking through **centralised exchanges**. First of all, crypto assets and CeFi are not suited well together from the fundamental standpoint.. It is also worth mentioning the economic aspect - by staking within an centralised entity, the user does not receive a corresponding token in return and, thus, loses the opportunity to perform any subsequent activity within DeFi or the same centralised entity, where assets were staked. Yes, APR, when staking on centralised exchanges might be higher, but with a significant amount aggregated within the centralised entity comes a huge potential influence to the ecosystem that was fundamentally designed decentralised.  
```

Comment:
CEX does not mean illiquid by default. Kraken are going to release liquid staking. soon as far as I know.